var/spack/repos/builtin/packages/easybuild/package.py

```diff
@@ -12,10 +12,16 @@ class Easybuild(PythonPackage):
     """
 
     homepage = 'http://hpcugent.github.io/easybuild/'
-    url      = 'https://pypi.io/packages/source/e/easybuild/easybuild-3.1.2.tar.gz'
+    url      = 'https://pypi.io/packages/source/e/easybuild/easybuild-4.0.0.tar.gz'
+    maintainers = ['boegel']
 
+    version('4.0.0', sha256='21bcc1048525ad6219667cc97a7421b5388068c670cabba356712e474896de40')
     version('3.1.2', 'c2d901c2a71f51b24890fa69c3a46383')
 
-    depends_on('py-easybuild-framework@3.1.2', when='@3.1.2', type='run')
-    depends_on('py-easybuild-easyblocks@3.1.2', when='@3.1.2', type='run')
-    depends_on('py-easybuild-easyconfigs@3.1.2', when='@3.1.2', type='run')
+    depends_on('python@2.6:2.8', when='@:3', type=('build', 'run'))
+    depends_on('python@2.6:2.8,3.5:', when='@4:', type=('build', 'run'))
+
+    for v in ['@4.0.0', '@3.1.2']:
```

Comment:
This is true for all versions, not just these...

Is there a way to express that more generally, or is the only way to list all known versions?

Reply:
Unfortunately, there is no way to make this true for all versions currently. Out of curiosity, does EasyBuild have a mechanism to do something like this?

Reply:
Well, since we always fix versions in easyconfig files, we don't really need this.

See for example https://github.com/easybuilders/easybuild-easyconfigs/blob/develop/easybuild/easyconfigs/e/EasyBuild/EasyBuild-3.9.4.eb